game/glshader.cc

```diff
@@ -49,6 +51,40 @@ void Shader::dumpInfoLog(GLuint id) {
 	std::clog << logmsg << std::flush;
 }
 
+void Shader::bindUniformBlocks() {
```

Comment:
We should add some comments here as I have no idea what an "uniform block" is.

Reply:
Uniforms organized in "interface blocks" which are essentially structs as you can see in the shaders. Cons: use more GPU memory, pros: shared state between all shaders so you only modify it once.